tests/functional/algos/test_algos.py

```diff
@@ -396,3 +467,41 @@ def test_parallel_workers(algorithm):
         param = best_trial._params[1]
         assert param.name == "x"
         assert param.type == "real"
+
+
+@pytest.mark.parametrize(
+    "algorithm",
+    branching_algorithm_configs.values(),
+    ids=list(branching_algorithm_configs.keys()),
+)
+def test_branching_algos(algorithm, tmp_path):
+    shutil.rmtree(tmp_path)
+    os.makedirs(tmp_path)
+
+    with OrionState() as cfg:  # Using PickledDB
```

Comment:
`cfg` seems unused here. 
```suggestion
    with OrionState():  # Using PickledDB
```

Also, would it be possible to use a fixture, to avoid indenting the entire body of this test?

Reply:
Oh yeah, all OrionState are unnecessary in this module. Thanks for pointing this out! Fixed in ca6a7cdd13ae1af5af33b8d4a6e1cb3331577882